src/main/java/io/strimzi/kafka/bridge/tracing/OpenTelemetryHandle.java

```diff
@@ -65,6 +70,7 @@ public String serviceName(BridgeConfig config) {
     @Override
     public void initialize() {
         System.setProperty("otel.metrics.exporter", "none"); // disable metrics
+        System.setProperty("otel.logs.exporter", "none"); // disable logs
```

Comment:
MAybe the comment can add some more explanation what logs and why are we disabling them?

Reply:
Resolved with https://github.com/strimzi/strimzi-kafka-bridge/pull/867/commits/8474fbb6d2b0cdbca3bee4237ca340a5702ee357